types/types.go

```diff
@@ -15,25 +17,23 @@ type ComMatrix struct {
 }
 
 type ComDetails struct {
-	Direction string `json:"direction"`
-	Protocol  string `json:"protocol"`
-	Port      string `json:"port"`
-	Namespace string `json:"namespace"`
-	Service   string `json:"service"`
-	Pod       string `json:"pod"`
-	Container string `json:"container"`
-	NodeRole  string `json:"nodeRole"`
-	Optional  bool   `json:"optional"`
+	Direction string `json:"Direction"`
+	Protocol  string `json:"Protocol"`
+	Port      int    `json:"Port,string"`
+	Namespace string `json:"Namespace"`
+	Service   string `json:"Service"`
+	Pod       string `json:"Pod"`
+	Container string `json:"Container"`
+	NodeRole  string `json:"Node Role"`
```

Comment:
I think the space here is wrong (in the `json:`), also is there a reason for the uppercase change in the rest?